test/integration/controller/issuer/issuer_suite_test.go

```diff
@@ -86,6 +100,7 @@ var _ = BeforeSuite(func() {
 	DeferCleanup(func() {
 		By("Stop test environment")
 		Expect(testEnv.Stop()).To(Succeed())
+		_ = os.RemoveAll(filepath.Dir(certificatePath))
```

Comment:
The code responsible for starting the Pebble server generates a certificate and private key on the fly in a temporary OS directory.
Instead of deleting the two files and directory individually we can clear them in one sweep.